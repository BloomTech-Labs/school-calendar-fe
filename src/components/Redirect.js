import React from 'react';
import { Redirect } from 'react-router-dom';

export default Redirect = () => {
  return (
    <div>
      <div>...redirecting</div>
      <Redirect to="/home" />
    </div>
  );
};
