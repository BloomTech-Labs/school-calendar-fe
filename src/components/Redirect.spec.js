import React from 'react';
import { shallow } from 'enzyme';
import Redirect from './Splash';


describe('Redirect', () => {
  it('should render correctly', () => {
    const component = shallow(<Redirect />);
  
    expect(component).toMatchSnapshot();
  });

  
});