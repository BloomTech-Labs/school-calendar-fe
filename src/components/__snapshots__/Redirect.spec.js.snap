// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Redirect should render correctly 1`] = `
<div
  className="splash"
>
  <Login />
</div>
`;
